implementations/java-mjs-validator/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM gradle:8.3.0-jdk11
```

Comment:
Please have a look at the other Java harnesses' Dockerfiles which have recently been fixed, this should use a multi-stage build to avoid having a very large resulting image. Although because this is not actually Java but Scala, you may still have what to figure out. Also please install from packages when available (I assume `sbt` should be, but you could also be using a base image that has Scala installed)